apps/oxlint/test/fixtures/fixes/plugin.ts

```diff
@@ -96,7 +98,7 @@ export default {
                   node,
                   fix(fixer) {
                     // Fixes can be in any order
-                    const range = [node.start, node.end];
+                    const range: Range = [node.start, node.end];
```

Comment:
I find it weird that adding this type annotation is required. Without it, TSC gives this error:

```
error TS2345: Argument of type 'number[]' is not assignable to parameter of type 'Range'.
Target requires 2 element(s) but source may have fewer.

fixer.replaceTextRange(range, 'er'),
                       ~~~~~
```

Type defs:

```ts
type Range = [number, number];

declare const FIXER: Readonly<{
  insertTextBeforeRange(range: Range, text: string): Fix;
  insertTextAfterRange(range: Range, text: string): Fix;
  replaceTextRange(range: Range, text: string): Fix;
  // ...
}>;
```

Is this an error in the type defs, or just how TypeScript works?